sdk/communication/azure-communication-chat/env-test.properties

```diff
@@ -1,9 +1,14 @@
 trouter.client.version="99116/0.0.0.0/"
 trouter.max.registration.ttls="3600"
-platform="SPOOL"
+platform="Android"
 platform.ui.version="0.0.0.0"
 
 trouter.hostname="go.trouter-int.skype.net"
 trouter.registration.hostname.and.base.path="edge.skype.net/registrar/testenv"
 trouter.application.id="AcsAndroid_test"
 trouter.template.key="AcsAndroid_Chat_test_1.2"
```

Comment:
Any reason why the above is `1.3` and this is `1.2`?

Reply:
I'll fix this.